weather_dl/download_pipeline/util.py

```diff
@@ -58,3 +59,12 @@ def ichunked(iterable: t.Iterable, n: int) -> t.Iterator[t.Iterable]:
             yield itertools.chain([first], it)
     except StopIteration:
         pass
+
+
+def copy(src: str, dst: str) -> None:
+    """Copy data via `gcloud storage cp`.
+
+    Here, we take advantage of GCP's faster storage transfer:
+    https://cloud.google.com/blog/products/storage-data-transfer/new-gcloud-storage-enables-super-fast-data-transfers/
+    """
+    subprocess.run(f'gcloud alpha storage cp {src!r} {dst!r}', shell=True, check=True)
```

Comment:
I suggest adding some error handling here.
```suggestion
    try:
        subprocess.run(f'gcloud alpha storage cp {src!r} {dst!r}', shell=True, check=True, capture_output=True)
    except subprocess.CalledProcessError as e:
        logger.error(f'Failed to copy file {src!r} to {dst!r} due to {e.stderr.decode("utf-8")}')
        raise
```

Reply:
Also should we also add retrying logic here or should we fail the complete pipeline if we are not able to upload a file ? WDYT ?

Reply:
Great thought! From the docs, it looks like `gcloud`/`gsutil` internally handles retries: https://cloud.google.com/storage/docs/gsutil/commands/cp#retry-handling

In my mind, when things fail here, they do so for a good reason. 

I like your log suggestions, I will adopt that :)